src/customEvent/rotate.js

```diff
@@ -17,13 +17,12 @@ eg.module("rotate", ["jQuery", eg, window, document], function($, ns, global, do
 	 *
 	 * @ko jQuery custom rotate 이벤트 지원
 	 * @name jQuery#rotate
-	 * @event
```

Comment:
@sculove 
Um.. It isn't event. I think that just information object like [jQueryUI](http://api.jqueryui.com/draggable/#event-create). Although it be ambiguous I think It is more `@info` than `@event`. Let's talk about this issues next week discuss.